tests/acceptance/yaml_test.py

```diff
@@ -49,118 +51,57 @@ def test_user_format_failure_case(testapp_with_base64):
                                 params={'required_arg': 'MQ'},)
 
 
+def validate_json_response(response, expected_dict):
+    assert response.headers['content-type'] == \
+           'application/json; charset=UTF-8'
+    assert json.loads(response.body.decode("utf-8")) == expected_dict
+
+
+def validate_yaml_response(response, expected_dict):
+    assert response.headers['content-type'] == \
+           'application/x-yaml; charset=UTF-8'
+    assert yaml.load(response.body) == expected_dict
+
+
+def fix_ref(ref, schema_format):
+    if schema_format == 'yaml':
+        return ref  # all refs are already yaml
+    return ref.replace('.yaml', '.%s' % schema_format)
+
+
+def walk_schema_for_refs(schema_item, schema_format):
```

Comment:
"walk_schema_for_refs" confused me...maybe "recursively_fix_refs"? Also is this just a helper to take a schema and make it look like it has external YAML/JSON refs easily? Might be worth a brief explanation in docstring or the consuming test